src/routes/scope-selection/ScopeSelectionScreen.tsx

```diff
@@ -73,6 +101,14 @@ const ScopeSelectionScreen = ({ navigation, route }: IntroScreenProps): JSX.Elem
     })
   }
 
+  const highlightText = (textArray: [string] | [string, string, string]): JSX.Element => (
+    <HighlightContainer>
+      <Text>{textArray[0]}</Text>
+      <BoldText>{textArray[1]}</BoldText>
+      <Text>{textArray[2]}</Text>
+    </HighlightContainer>
+  )
```

Comment:
Just as a note. This only highlights the first occurence of the match. In this case this should be fine though :).

Reply:
Yes, I decided that's fine here :)